docs/Privacy/Start-Pantheon-Privacy.md

```diff
@@ -0,0 +1,334 @@
+       "chainID": 1981
+   },
+  "nonce": "0x42",
+  "gasLimit": "0x1000000",
+  "difficulty": "0x10000",
+  "alloc": {
+    "fe3b557e8fb62b89f4916b721be55ceb828dbd73": {
+      "privateKey": "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+      "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+      "balance": "0xad78ebc5ac6200000"
+    },
+    "f17f52151EbEF6C7334FAD080c5704D77216b732": {
+      "privateKey": "ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f",
+      "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+      "balance": "90000000000000000000000"
+    }
+  }
+}
+```
+
+!!! warning
+    Do not use the accounts in the genesis file above on mainnet or any public network except for testing.
+    The private keys are displayed so the accounts are not secure. 
+    
+### 3. Get Public Key of First Node
+
+To enable nodes to discover each other, a network requires one or more nodes to be bootnodes. 
+For this private network, we will use Node-1 as the bootnode. This requires obtaining the public key for the [enode URL](../Configuring-Pantheon/Node-Keys.md#enode-url). 
+
+In the `Node-1` directory, use the [`public-key` subcommand](../Reference/Pantheon-CLI-Syntax.md#public-key) to write 
+the [node public key](../Configuring-Pantheon/Node-Keys.md#node-public-key) to the specified file (`publicKeyNode1` in this example):
+
+```bash tab="MacOS"
+pantheon --data-path=Node-1-data-path --genesis-file=../privateNetworkGenesis.json public-key export --to=Node-1-data-path/publicKeyNode1
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-1-data-path --genesis-file=..\privateNetworkGenesis.json public-key export --to=Node-1-data-path\publicKeyNode1
+```
+
+!!!note
+    The [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) and [`--genesis-file`](../Reference/Pantheon-CLI-Syntax.md#genesis-file) 
+    options are not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md). 
+    Use a bind mount to [specify a configuration file with Docker](../Getting-Started/Run-Docker-Image.md#custom-genesis-file)
+    and volume to [specify the data directory](../Getting-Started/Run-Docker-Image.md#data-directory).
+
+Your node 1 directory now contains: 
+```bash
+├── Node-1
+    ├── Node-1-data-path
+        ├── database
+        ├── key
+        ├── publicKeyNode1
+```
+      
+The `database` directory contains the blockchain data. 
+
+### 4. Start First Node as Bootnode 
+
+Start Node-1:
+
+```bash tab="MacOS"
+pantheon --data-path=Node-1-data-path --genesis-file=../privateNetworkGenesis.json --bootnodes 
+--miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-enabled 
+--host-whitelist=* --rpc-http-cors-origins="all" --privacy-enabled=true --privacy-precompiled-address=125
+--privacy-url=127.0.0.1 --rpc-http-api=EEA    
```

Comment:
Does the `privacy-url` also need to include the port?  eg, "http://127.0.0.1:8888/"

Is the `privacy-public-key-file` option also required to specify the public key of the Orion node?